Arrays and Pointers/arrays.cpp

```diff
@@ -12,7 +12,7 @@
 #include <iomanip>
 #include <set>
 #include <map>
-
+// #include<bits/stdc++.h>  // By adding this single header file we can add all standard header files
```

Comment:
If we're adding ```#include<bits/stdc++.h>``` please remove rest of header files

Reply:
Yes sir, I have removed other header files. Thank you !